lib/m3u.rb

```diff
@@ -0,0 +1,22 @@
+module FormatParser
+  class M3U
```

Comment:
I would name this `Text` instead, to make it in line with the others.

Reply:
I named it M3U considering we will add more "m3u specific" directives here to store information. If we agree that the information we currently have is suffice, I can rename it to Text

Reply:
I think he means `M3u` (not using an upper case)

Reply:
No, I see this in line with `audio`, `video`, `document` etc. M3U is for Text what MP4 is for audio, as I see it.

Reply:
hmm, it makes sense!